frontend/src/exams/edit-group-exam-modal.vue

```diff
@@ -44,19 +44,19 @@
                           lang="en"></DatePicker>
             </b-form-group>
           </b-col>
-          <b-col cols="6" v-if="role_code !== 'GA' && role_code !== 'LIAISON'">
+          <b-col cols="6" v-if="role_code !== 'GA' && liaison_designate !== 1">
```

Comment:
This would be a good spot for a getter as well. `isGaOrDesignate` or something

Reply:
I'll cover this in a separate PR